upup/pkg/fi/cloudup/openstacktasks/instance.go

```diff
@@ -45,6 +46,7 @@ type Instance struct {
 	Metadata         map[string]string
 	AvailabilityZone *string
 	SecurityGroups   []string
+	FloatingIP       *FloatingIP
```

Comment:
`Find()` needs to fill in this field.

Reply:
I agree that at some point Find should fill in this. But right now, there is a `CheckChanges` that only returns true when the instance doesn't exist. So an instance will basically never be updated once created. That is, if I understand fi.task correctly.

One can easily run into an issue where floating IP and instance is created, but association fails for some reason, and it would be nice to have kops detect and amend that. But that's the case for a number of things here. Had a plan to clean that part up later as well.